app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  before_action :authenticate_user!
```

Comment:
- [ ] @team great work implementing the changes requested, however as the previous reviewer suggested kindly add  ` before_action :authenticate_user!` to the `UsersController` class in your controllers' folder and not the user model.